app/src/main/java/com/skydoves/pokedex/repository/MainRepository.kt

```diff
@@ -44,7 +45,8 @@ class MainRepository @Inject constructor(
     page: Int,
     onStart: () -> Unit,
     onSuccess: () -> Unit,
-    onError: (String?) -> Unit
+    onError: (String?) -> Unit,
+    onDone: () -> Unit
```

Comment:
How do you think just change `onSuccess` to `onComplete` and remove the `onDone`?
I think actually the role of the `onSuccess` is only making the `isLoading` false.

Reply:
yes. We can do that. the onSuccess is redundant.